tuf/utils/utils.go

```diff
@@ -150,3 +151,37 @@ func ConsistentName(role string, hashSha256 []byte) string {
 	}
 	return role
 }
+
+// ErrWrongLength indicates the length was different to that expected
+var ErrWrongLength = errors.New("wrong length")
+
+// ErrWrongHash indicates the hash was different to that expected
+type ErrWrongHash struct {
+	Type     string
+	Expected []byte
+	Actual   []byte
+}
+
+// Error implements error interface
+func (e ErrWrongHash) Error() string {
```

Comment:
I think all these errors can also be removed, since I can't seem to find anywhere they're being used?